### DISCUSSION TITLE
Keys used for letter input should not respond to shortcut keys in chat

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#28162] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/34775378/ed842066-d13d-4cf1-9c92-18d6dfbf3340)

also for beatmap search

**Comment by bdach** [comment]
Doesn't happen on linux at least. May be windows-specific, will need to verify there.

**Reply by bdach** [reply]
I can't reproduce on windows either using japanese IME.

Please provide full reproduction steps.

**Reply by cdwcgt** [reply]
I'm inconvenient to using the official build for testing for now (I need to wait next release for database)
but I can reproduce it on the current master (badda23dde8dc9cb42ed40824b0b46391fa0e1b5)

note that debug behavior is a little different from release one?

https://github.com/ppy/osu/assets/34775378/6660f28f-6cc4-4432-b4d5-7f02901cd401

test official release version with empty database but I can't seem to reproduce it
I will find time to test it with the official release version

**Reply by cdwcgt** [reply]
https://github.com/ppy/osu/assets/34775378/a2a68ae7-4271-44fd-ac08-ae0b80fa8bc0

yes, it can easily reproduce in 2024.519.1

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/28261
